visualization/app/codeCharta/util/treeMapHelper.ts

```diff
@@ -100,10 +100,10 @@ function buildNodeFrom(
 		color: getBuildingColor(squaredNode.data, s, isDeltaState, flattened),
 		incomingEdgePoint: getIncomingEdgePoint(width, height, length, new Vector3(x0, z0, y0), s.treeMap.mapSize),
 		outgoingEdgePoint: getOutgoingEdgePoint(width, height, length, new Vector3(x0, z0, y0), s.treeMap.mapSize)
-	}
+	} as Node
```

Comment:
I encountered another line of code where you defined a return type instead of declaring what the type is during return.
We should choose one. I'd rather keep the return type.